run_google_cloud_tests.sh

```diff
@@ -0,0 +1,3 @@
+set -e
+
+bash tools/run_gpu_tests.sh
```

Comment:
Similar to a little while ago, this requires Googlers to make a quick change on the GPU test script. Marking blocked until that is complete

Reply:
Thanks! Hopefully it'll be general enough this time :)